core/src/main/java/com/sequenceiq/cloudbreak/converter/StackToTemplatePreparationObjectConverter.java

```diff
@@ -392,6 +394,17 @@ private boolean isCloudStorageConfigured(Stack source) {
 
     private GeneralClusterConfigs calculateGeneralClusterConfigs(Stack source, Cluster cluster) {
         GeneralClusterConfigs generalClusterConfigs = generalClusterConfigsProvider.generalClusterConfigs(source, cluster);
+        List<SdxClusterResponse> clients = sdxClientService.getByEnvironmentCrn(source.getEnvironmentCrn());
```

Comment:
why we need the datalake side?

Reply:
Well, in case of a medium duty cluster Kafka will have 3 nodes where we'll need to enable racks so I thought we would likely to use this on both DL and DH sides.

Reply:
I'll try to move this logic to the provider as I realized that probably just the UI sends all the instance groups with multiple subnets. Will get back a bit later.

Reply:
I removed the datalake call.